### ISSUE TITLE
Ensure Quarkus in JVM-mode works on Eclipse OpenJ9

### ISSUE THREAD

**Issue by DanHeidinga** [quarkusio/quarkus#6538]
### Description
The Eclipse OpenJ9 JVM is an alternative to the Hotspot JVM.  It provides a cross-platform Java environment that includes support for the P (Power) & Z (s390) architectures.  The OpenJ9 team is interested in ensuring that Quarkus in JVM mode runs well on OpenJ9 and provides a great user experience.

This issue is to help coordinate any issues found while testing Quarkus on OpenJ9 on P, Z and other platforms.

### Analysis
OpenJ9 will need to run all Quarkus tests on P, Z, and X to ensure it works well.

### Tasks
- [ ] Run Quarkus tests at AdoptOpenJDK on both OpenJ9 and Hotspot.  Currently scheduled weekly. https://ci.adoptopenjdk.net/view/Test_external_weekly/
- [ ] Extend testing at AdoptopenJDK to P & Z architectures and JDK 11: https://github.com/AdoptOpenJDK/openjdk-tests/issues/1552
- [ ] Identify and address any issues found running tests
- [X] LogManager issue https://github.com/eclipse/openj9/issues/8237 - resolved for JDK8 in latest release and fix being delivered for jdk11 https://github.com/eclipse/openj9/pull/8319
- [X] Documentation tests fail https://github.com/eclipse/openj9/issues/8303 - only happens on old releases of OpenJ9.  Doesn't occur with the latest releases.
- [ ] Mongodb tests incorrectly detect P & Z linux as 32bit on both Hotspot & OpenJ9 - https://github.com/quarkusio/quarkus/issues/3423
- [ ] io.quarkus.it.kafka.streams.KafkaStreamsTest.testKafkaStreams fail to locate JNI library on Z #6667

**Comment by smlambert** [comment]
For KafkaStreamsTest failure, raised https://github.com/quarkusio/quarkus/issues/6667

**Comment by Sanne** [comment]
@DanHeidinga hi, a suggestion:

I've used flapdoodle myself in the past while working on Hibernate OGM's support for MongoDB, and eventually we decided to get rid of it as it was too problematic to maintain. It might be easier to convert such integration test to use a container image, and start it like we do with various RDBMs integration tests. Not least it would test "the real thing", which is always better.

**Comment by gustavolira** [comment]
Are there any progress on this issue? Does Quarkus currently guarantee full compatibility with openj9?

**Comment by DanHeidinga** [comment]
TLDR: Quarkus on OpenJ9 - especially on P & Z linux - works just fine JVM mode.  If you find something that doesn't, please report it here.

Thanks for the reminder to reflect the progress in this issue.  As of Quarkus 1.7, the zLinux support is solid and even included in Red Hat's supported configurations [1].  Similar levels of quality are present on pLinux as well and I fully expect things to work on both platforms.

Additionally, a lot of Quarkus tests - fv tests, quick starts, openshift - are run at the AdoptOpenJDK project using both Hotspot and OpenJ9 across a wide variety of platforms.  Any issues found there are reported back to the upstream projects to ensure that they are addressed as quickly as possible.

If you've found something that doesn't work when running OpenJ9, I'd appreciate knowing about it.  Opening issues and linking them here is a great way to report any concerns running on OpenJ9.

[1] https://access.redhat.com/articles/4966181

**Comment by DanHeidinga** [comment]
Given the previous comment and the lack of reported issues with OpenJ9 in the interim, closing this out
